src/components/organism/AddPoolLiquidity/index.tsx

```diff
@@ -121,29 +120,16 @@ const AddPoolLiquidity = () => {
   const handlePool = async () => {
     try {
       if (api) {
-        if (!poolExists) {
-          await createPool(
-            api,
-            selectedTokenB.assetTokenId,
-            selectedAccount,
-            nativeTokenValue,
-            assetTokenValue,
-            nativeTokenWithSlippage.tokenValue.toString(),
-            assetTokenWithSlippage.tokenValue.toString(),
-            dispatch
-          );
-        } else {
-          await addLiquidity(
-            api,
-            selectedTokenB.assetTokenId,
-            selectedAccount,
-            nativeTokenValue,
-            assetTokenValue,
-            nativeTokenWithSlippage.tokenValue.toString(),
-            assetTokenWithSlippage.tokenValue.toString(),
-            dispatch
-          );
-        }
+        await addLiquidity(
```

Comment:
it is not a blocker it is more like a suggestion. If u have more than 2 or 3 properties in the function, make them as an object. Right now u can easily switch places between properties on the other side u will get a wrong value.
Eg: awit test (api, selectedTokenB.assetTokenId, selectedAccount, nativeTokenValue, assetTokenValue, nativeTokenWithSlippage.tokenValue.toString(), assetTokenWithSlippage.tokenValue.toString(), dispatch)
   u can put await test({api, selectedTokenB: selectedTokenB.assetTokenId, etc...})